main.js

```diff
@@ -0,0 +1,111 @@
+const game = {
```

Comment:
this is a BIG one game object, the best practice is to use a class with method and properties.